backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1758,3 +1758,61 @@ databaseChangeLog:
             columns:
               - column:
                   name: test_result_delivery
+  - changeSet:
+      id: add-no-phi-views
+      author: josh@skylight.digital
+      comment: Add views for some tables that omit columns containing PHI.
+      changes:
+        - createView:
+            viewName: api_user_no_phi_view
+            remarks: A subset of the api_user table with columns containing PHI removed.
+            replaceIfExists: true
+            fullDefinition: false
+            selectQuery: SELECT internal_id, created_at, updated_at, last_seen, is_deleted FROM ${database.defaultSchemaName}.api_user
+        - createView:
+            viewName: provider_no_phi_view
+            remarks: A subset of the provider table with columns containing PHI removed.
```

Comment:
Nitpick: there is no PHI in this table (see further discussion below).

Reply:
@TimBest thoughts?